src/extension/index.ts

```diff
@@ -42,8 +42,7 @@ export async function activate(context: ExtensionContext) {
     disposables.push(commands.registerCommand('sarif.showPanel', () => panel.show()));
 
     // General Activation
-    const refreshDiagnostics = activateDiagnostics(disposables, store, baser);
```

Comment:
>activateDiagnostics [](start = 31, length = 19)

Why are we no longer doing this?

Reply:
refreshDiagnostics was a "hook" to allow CodeActions to call. Removing the feature removes the need for this.